### DISCUSSION TITLE
推荐一个没人看的MC 职业pvp地图

### DISCUSSION THREAD

**Discussion by SuperBigDummy** [share121/inter-knot#1816] [其他]
![a5f86275e23b672694f613231fe7e8a0285815215](https://github.com/user-attachments/assets/3f26ac3a-a0e5-466f-a741-9c4516bcbdd4)
（好吧我就是来试试发帖功能的）
链接放这了：[https://www.bilibili.com/video/BV1xUiwevEmU/?share_source=copy_web&vd_source=4d3c7cf716edab0cb72650245332d5a2](url)
自己去看吧

**Comment by gaojiaobei** [comment]
111，链接直接点不开

**Reply by SuperBigDummy** [reply]
这个委托我接不了.建议复制后打开

**Comment by 834398220** [comment]
链接点不开

**Comment by SuperBigDummy** [comment]
建议就链接打不开这件事发一个帖求助[手动doge]
